hoot-core/src/main/cpp/hoot/core/conflate/network/NetworkMatch.cpp

```diff
@@ -135,8 +135,8 @@ bool NetworkMatch::isConflicting(const Match& other, const ConstOsmMapPtr& /*map
     {
       const pair<ElementId, ElementId>& jp = *jt;
 
-      if (ip.first == jp.first || ip.second == jp.first ||
-        ip.second == jp.first || ip.second == jp.second)
+      if (ip.first == jp.first  || ip.first == jp.second ||
+          ip.second == jp.first || ip.second == jp.second)
```

Comment:
Here if `ip.first == jp.second` previously it wasn't being set as a conflict.  This was a logic error.